src/constants.json

```diff
@@ -56,6 +56,7 @@
     "DEFAULT_ENDPOINT" : "https://prebid.adnxs.com/pbs/v1/auction",
     "SRC" : "s2s",
     "ADAPTER" : "prebidServer",
-    "SYNC_ENDPOINT" : "https://prebid.adnxs.com/pbs/v1/cookie_sync"
+    "SYNC_ENDPOINT" : "https://prebid.adnxs.com/pbs/v1/cookie_sync",
+    "SYNCED_BIDDERS_KEY": "pbjsSyncs"
```

Comment:
@mkendall07 pointed out in another one of my PRs that this `constants` file does more harm than good, now that we can `export const SYNCED_BIDDERS_KEY = "pbjsSyncs"` somewhere in the code. I think he's got a good point.